### ISSUE TITLE
Problem with new search_vector field on Media

### ISSUE THREAD

**Issue by bruvellu** [nelas/cifonauta#233]
![search_vector_error](https://github.com/bruvellu/cifonauta/assets/361179/3f036e73-c798-4351-8451-9840945cbabc)

The relevant part of the code on `meta.models.py`:

```
        self.search_vector = SearchVector('title_pt_br', weight='A', config='portuguese_unaccent') + \
                             SearchVector('title_en', weight='A', config='english') + \
                             SearchVector('caption_pt_br', weight='B', config='portuguese_unaccent') + \
                             SearchVector('caption_en', weight='B', config='english')
```

Apparently, this is caused by the `weight` option, which uses an F object (internally):

https://stackoverflow.com/questions/47242639/how-to-add-an-arrayfield-to-a-searchvector-in-django/53187585#53187585

Updated the code to use `Value` as described in the post above:

```
    def save(self, *args, **kwargs):

        # Update search vector field on save
        self.search_vector = SearchVector(Value(self.title_pt_br), weight='A', config='portuguese_unaccent') + \
                             SearchVector(Value(self.title_en), weight='A', config='english') + \
                             SearchVector(Value(self.caption_pt_br), weight='B', config='portuguese_unaccent') + \
                             SearchVector(Value(self.caption_en), weight='B', config='english')
```

**Comment by bruvellu** [comment]
Fixed in 73896d24c27bf64f708cefcff7cadb33810f1e94

**Comment by bruvellu** [comment]
Moved the `search_vector` update to a `post_save` signal 33d12366c1c9d8f67f294993f65663499cf3dc7f using a slightly hacky query set `update` method. But it works smoothly. For some reason, the `pre_save` didn't work as I had intended, as the `search_vector` field was only updated after two saves.
